packages/web-sockets/config/redis.js

```diff
@@ -0,0 +1,13 @@
+// External dependencies
+const dotenv = require('dotenv');
+
+// Internal dependencies
+const path = require('./environment');
+
+dotenv.config({ path });
+
+module.exports = {
+  host: process.env['WEB.SOCKET.REDIS.HOST'],
```

Comment:
nit-pick: Env vars are usually in underscore case, can you change them to be underscore case?

Reply:
Well I use  dot notation  because eslint suggest that, but yeah I'll make the change